docs/user-guide/microgrid-concepts.md

```diff
@@ -0,0 +1,128 @@
+# Microgrid concepts
+
+A microgrid is a local electrical grid that connects a set of
+electrical components together.  They are often built around a passive
+power consumer, to supplement the electricity consumed from the public
+grid, with on-site power generation or storage systems.
+
+Microgrids can also function in island-mode, without a grid
+connection, or without a local power consumer, but they have to have
+at least one of the two, to be meaningful.
+
+## Frequenz SDK Microgrid Model
+
+The SDK aims to provide an abstract model of the microgrid that allows
+for high-level interactions with microgrid components, without having
+to deal with, or even be aware of location specific details like:
+
+- where the meters are placed,
+- how many batteries,
+- whether there's a grid connection or a passive consumer,
+- what models the inverters are, etc.
+- whether components are having downtimes, because metrics and limits
+  get adjusted automatically when components are having downtimes.
+
+Users of the SDK can develop applications around this interface once
+and deploy anywhere, and the SDK will take care of translating the
+requests and instructions to correspond to the specific microgrid
+configurations.
+
+``` mermaid
+flowchart LR
+
+subgraph Left[Measurements only]
+direction LR
+  grid["Grid Connection"]
+  consumer["Consumer"]
+  pv["PV Arrays"]
+  chp["CHP"]
+end
+
+junction(( ))
+
+subgraph Right[Measurements and control]
+direction LR
+  bat["Batteries"]
+  ev["EV Chargers"]
+end
+
+grid --- junction
+consumer --- junction
+pv --- junction
+chp --- junction
+
+junction --- bat
+junction --- ev
+```
+
+## Grid
+
+This refers to a microgrid's connection to the external Grid.  The
+power flowing through this connection can be streamed through
+[`grid_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.grid_power].
+
+In locations without a grid connection, this method would still be
+available, and would stream zero values.
+
+## Consumer
+
+This is the main power consumer at the site of a microgrid, and often
+the load the microgrid is built to support.  The power drawn by the
+consumer is available through
+[`consumer_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.consumer_power]
+
+If there is no consumer at a site, this method would stream zero values.
+
+## Producers: PV Arrays, CHP
+
+The total PV power production in a microgrid can be streamed through
+[`pv_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.pv_power]
+, and similarly the total CHP production in a site can be streamed
+through
+[`chp_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.chp_power].
+And total producer power is available through
+[`producer_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.producer_power].
+
+And just like the other methods, if components of either of these
+types are not available in a microgrid, the corrensponding methods
+will stream zero values.
```

Comment:
```suggestion
Similarly to the other methods, if components of either of these
types are unavailable in a microgrid, the corresponding methods
will stream zero values.
```

Reply:
Rewrote with "As is the case with the other methods, ..."